crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,119 @@ x-----x
+# to explore, and unique and random on the inside. 
+
+NAME:   shapermc_ecumenical_altar_mp_vishouse
+TAGS:   transparent no_monster_gen
+SUBST:  - = --.G
+SUBST:  G = GGG-
+COLOUR: - = brown
+FTILE:  -G = floor_rough_brown
+COLOUR: G = lightgrey
+TILE:   G = dngn_crumbled_column
+: set_feature_name("granite_statue", "broken pillar")
+: ecumenical_altar_setup(_G)
+
+MAP
+.............                                               
+.....xxxxx..x                                               
+..xxx---xG--x                                               
+xx------xxxxx                                               
+x--xxx--x---x                                               
+x--x_x------x                                               
+x--b-x--xx+xx                                               
+x--+-x--x---x                                               
+x--b-x--xGGGx                                               
+xxxxxx.xxxxxx                                               
+.............  
+ENDMAP
+
+
+NAME:   shapermc_ecumenical_altar_mp_highgroup
+TAGS:   transparent no_monster_gen
+SUBST:  - = --.G
+SUBST:  G = GGG-
+COLOUR: - = brown
+FTILE:  -G = floor_rough_brown
+COLOUR: G = lightgrey
+TILE:   G = dngn_crumbled_column
+: set_feature_name("granite_statue", "broken pillar")
+: ecumenical_altar_setup(_G)
+NSUBST: ^ = 4=^. / 4=^..
+KFEAT:  ^ = web trap
+
+MAP
+xxxxxx-xxxxxx                                             
+xG^--x-x--^Gx                                             
+xxx-xx_xx-xxx                                             
+x-----------x                                             
+x^x---.---x^x                                             
+xG^--...--^Gx                                             
+xxxx.....xxxx
+ENDMAP
+
+NAME:   shapermc_ecumenical_altar_mp_urbaneastern
+TAGS:   transparent no_monster_gen
+SUBST:  - = --.G
+SUBST:  G = GGG-
+COLOUR: - = brown
+FTILE:  -G = floor_rough_brown
+COLOUR: G = lightgrey
+KMONS:  p = plant
+KMONS:  f = fungus
+NSUBST: ' = 8:pf / 12=ppf... / .
+TILE:   G = dngn_crumbled_column
+: set_feature_name("granite_statue", "broken pillar")
+: ecumenical_altar_setup(_G)
+
+MAP
+...............                                             
+.xxxxxxxxxxxxx.                                             
+..x-'-x'x-'-x..                                             
+..x-x-x'x-x-x..                                             
+..''''''''''''.                                             
+..x-x-x'x-x-x..                                             
+..x-'-x_x-'-x..                                             
+..x-x-x'x-x-x..                                             
+..'''''''''''..                                             
+..x-x-x'x-x-x..                                             
+..x-'-x'x-'-x..                                             
+.xxxxxxxxxxxxx.                                             
+............... 
+ENDMAP
+
+NAME:   shapermc_ecumenical_altar_mp_highgroup
```

Comment:
This should be shapermc_ecumenical_altar_mp_ritualfountain ... trying to figure out how to edit this :) Hopefully this will resolve before you see the note.